Auth0/Authentication.swift

```diff
@@ -83,6 +83,7 @@ public protocol Authentication: Trackable, Loggable {
      - returns: authentication request that will yield Auth0 User Credentials
      - seeAlso: Credentials
      */
+    @available(*, deprecated, message: "see login(usernameOrEmail username: String, password: String, realm: String, audience: String?, scope: String?)")
     // swiftlint:disable:next function_parameter_count
     func login(usernameOrEmail username: String, password: String, multifactorCode: String?, connection: String, scope: String, parameters: [String: Any]) -> Request<Credentials, AuthenticationError>
```

Comment:
Add info of what grant type needs to be enabled and a link to the docs about grant type